migrations/20210817115335_create_db_inits.postgres.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE contents (
+   id bigserial PRIMARY KEY,
+   title varchar(150) NOT NULL,
+   details TEXT NOT NULL,
+   created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+   updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
```

Comment:
`updated_at` should have default as null.

Reply:
This is old PR, please close it and don't merge it